.github/workflows/ci.yml

```diff
@@ -3,12 +3,16 @@ name: CI
 on: [push, pull_request]
 jobs:
   build:
-    runs-on: ubuntu-latest
     permissions:
       contents: read
     strategy:
       matrix:
         node-version: [14.x, 16.x, 18.x]
+        os: [ubuntu-latest, windows-latest]
+        exclude:
+          - os: windows-latest
+            node-version: 14.x
```

Comment:
exclude it altogether please, we don't need to support node 14 anymore

Reply:
That'd be a semver-major bump. I don't mind doing so in this module, maybe we can drop v16 too.